# All Action message should have a goal_id
actionlib_msgs/GoalID goal_id
string name
string data